src/Approximations/template_directions.jl

```diff
@@ -59,45 +62,97 @@ function isbounding(ad::AbstractDirections)
     return false
 end
 
+"""
+    isnormalized(ad::AbstractDirections)
+
+Returns whether the given template directions is normalized with respect to the
+2-norm.
+
+### Input
+
+- `ad` -- template directions
+
+### Output
+
+`true` if the 2-norm of each element in `ad` is one and `false` otherwise.
+"""
+function isnormalized(ad::AbstractDirections) end
```

Comment:
```suggestion
function isnormalized(ad::AbstractDirections)
    return false
end
```
This defines the function for all subtypes.

Reply:
Yes, that sets an arbitrary default value.

Reply:
It is not arbitrary. It is the pessimistic case. In applications you hope for the positive case.